UI/src/app/dashboard/controllers/admin.js

```diff
@@ -326,5 +326,81 @@
                 }
             );
         }
+ 
+//Configuration settings functionality starts here
+        dashboardData.getGeneralConfig().then(processGeneralConfigResponse);
+        function processGeneralConfigResponse(data){
+            console.log(data)
```

Comment:
@dhivahar19  Remove logging here, data may contain password or token information

Reply:
It has been removed